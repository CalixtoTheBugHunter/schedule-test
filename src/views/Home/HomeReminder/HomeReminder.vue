<template>
    <section class="HomeReminder">
        <base-input :value="title" placeholder="Untitled" />
        <base-datepicker :value="date" placeholder="Date" />
        <base-input :value="city" placeholder="City" />
        <base-input :value="color" placeholder="Color" />
        <base-button label="Save reminder" />
    </section>
</template>

<script>
import { BaseButton, BaseDatepicker, BaseInput } from '@/components/atoms'

export default {
    components: { BaseButton, BaseDatepicker, BaseInput },
    name: 'HomeReminder',
    data() {
        return {
            title: '',
            date: '',
            city: '',
            color: ''
        }
    }
}
</script>

<style lang="scss" scoped>
.HomeReminder {
    color: $color-primary;
}
</style>