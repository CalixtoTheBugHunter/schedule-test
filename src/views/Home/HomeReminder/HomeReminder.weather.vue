<template>
    <p class="ReminderWeather">
        {{ value.main }}
        <img :src="`http://openweathermap.org/img/w/${value.icon}.png`" />
    </p>
</template>

<script>
export default {
    name: 'ReminderWeather',
    props: {
        value: {
            type: Object,
            default: () => {}
        }
    }
}
</script>

<style lang="scss" scoped>
.ReminderWeather {
    color: $color-primary;
}
</style>