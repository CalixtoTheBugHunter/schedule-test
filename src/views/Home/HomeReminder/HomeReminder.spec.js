/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import HomeReminder from './HomeReminder'

describe('HomeReminder', () => {
  const WRAPPER = shallowMount(HomeReminder)

  it('Component exists', () => {
    expect(WRAPPER.exists()).toBe(true)
  })

  it('Component not changed', () => {
    expect(WRAPPER.html()).toMatchSnapshot()
  })
})