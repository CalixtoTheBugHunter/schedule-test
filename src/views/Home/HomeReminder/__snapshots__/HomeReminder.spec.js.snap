// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeReminder Component not changed 1`] = `
<section class="HomeReminder">
  <base-input-stub value="" placeholder="Untitled"></base-input-stub>
  <base-datepicker-stub value="" placeholder="Date"></base-datepicker-stub>
  <base-input-stub value="" placeholder="City"></base-input-stub>
  <base-input-stub value="" placeholder="Color"></base-input-stub>
  <base-button-stub label="Save reminder"></base-button-stub>
</section>
`;
